x-pack/plugins/enterprise_search/public/applications/workplace_search/views/overview/recent_activity.tsx

```diff
@@ -15,7 +15,7 @@ import { FormattedMessage } from '@kbn/i18n/react';
 import { ContentSection } from '../../components/shared/content_section';
 import { sendTelemetry } from '../../../shared/telemetry';
 import { KibanaContext, IKibanaContext } from '../../../index';
-import { getSourcePath } from '../../routes';
+import { SOURCE_DETAILS_PATH, getContentSourcePath as sourcePath } from '../../routes';
```

Comment:
I'm willing to drop this comment if you disagree, but I generally prefer not to rename the variables in imports. If we do it here and forget to do it somewhere else, we would basically have one variable under two names. Also, it makes searching the codebase a bit harder.